### DISCUSSION TITLE
mirai3包引用异常问题无注解

### DISCUSSION THREAD

**Discussion by Xbai11** [simple-robot/Discussions#189] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai

### 疑问描述

为什么3.x版我的相关注解在idea里都会爆红（但是不影响运行但没有代码提示）部分类没有，接发消息无示例类

**Comment by ForteScarlet** [comment]
参考 https://simple-robot-library.github.io/simbot3-website/faq/%E5%8C%85%E5%BC%95%E7%94%A8%E5%BC%82%E5%B8%B8/

道理差不多意思
